duo_client/client.py

```diff
@@ -244,20 +321,25 @@ def api_call(self, method, path, params):
                     self.host,
                     path,
                     now,
-                    self.sig_version,
+                    sig_version,
                     params,
                     body=body,
-                    digestmod=self.digestmod)
+                    digestmod=digestmod,
+                    additional_headers=additional_headers)
         headers = {
             'Authorization': auth,
             'Date': now,
         }
 
+        if sig_version == 5:
+            for k, v in additional_headers.items():
+                headers[k] = v
```

Comment:
Should the additional_headers also go to the regular headers?

If so, do they need to be sorted?

Reply:
On another thought, we might not need this to be sorted because server side sort header with `canon_x_duo_headers` already. So client should able to send over header in any order.